youtube_dl/extractor/geeksandsundry.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+import re
+
+
+class GeeksAndSundryIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?geekandsundry\.com/(?P<title>.+)'
+    _TEST = {
+        'url': 'http://www.geekandsundry.com/tabletop-bonus-wils-final-thoughts-on-dread/',
+        'md5': '02206df2e7a1805349a75af8df396222',
+        'info_dict': {
+            'id': 'tabletop-bonus-wils-final-thoughts-on-dread/',
+            'ext': 'mp4',
+            'title': 'TableTop Bonus! Wil\u2019s Final Thoughts on Dread | Geek and Sundry',
+        }
+    }
+
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        page = mobj.group('title')
```

Comment:
Usually variables named ```page``` are for storing downloaded data (webpage, API page, etc.). This variable is usually named as ```video_id```, ```media_id```, etc. Also, it's better to use ```_match_id``` function here.